model/email.py

```diff
@@ -44,6 +44,14 @@ def has_credentials(self):
             return True
         return False
 
+    def blacklist(self):
+        return self.raw.get('blacklist')
```

Comment:
default to empty collection of absent - problem solved, and one codepath less

also, as discussed: add a method doing the filtering, instead